package.json

```diff
@@ -42,14 +42,14 @@
   },
   "devDependencies": {
     "@67p/ember-service-worker-hubot-web-push-notifications": "^1.0.1",
+    "@csstools/postcss-sass": "^4.0.0",
     "@ember/optional-features": "^2.0.0",
     "@ember/render-modifiers": "^1.0.2",
     "@ember/test-helpers": "^2.2.5",
     "@glimmer/component": "^1.0.1",
     "@glimmer/tracking": "^1.0.4",
-    "autoprefixer": "^7.2.6",
+    "autoprefixer": "^10.2.5",
```

Comment:
Is this library still needed? It looks like all references to it have been removed from `ember-cli-build.js`.

Reply:
You're right! Tailwind apparently comes with autoprefixer by default.